src/scss/mixins/_toggle.scss

```diff
@@ -7,40 +7,42 @@
 /// @see {mixin} oFormsCheckboxToggleInverse - An inversed toggle style is avalible.
 /// @output Styles for toggle interface which is backed by a checkbox input.
 @mixin oFormsCheckboxToggle($class: 'o-forms') {
-	.#{$class}__label {
-		@include _oFormsRadioCheckboxLabel();
-		padding-top: $_o-forms-radiocheckbox-default-size / 2 - 10px; // To align the text with the center of the toggle checkbox
-		padding-left: ($_o-forms-radiocheckbox-default-size * 1.6) + $_o-forms-label-padding-left;
-		min-height: $_o-forms-radiocheckbox-default-size;
-		&:before {
-			position: absolute;
-			content: '';
-			background-color: oColorsGetColorFor(o-forms-field-label, text);
-			top: 0;
-			left: 0;
-			width: $_o-forms-radiocheckbox-default-size * 1.6;
-			height: $_o-forms-radiocheckbox-default-size;
-			border-radius: ($_o-forms-radiocheckbox-default-size * 1.6);
+	@include oBrandConfigureFor('o-forms') {
+		.#{$class}__label {
+			@include _oFormsRadioCheckboxLabel();
+			padding-top: $_o-forms-radiocheckbox-default-size / 2 - 10px; // To align the text with the center of the toggle checkbox
+			padding-left: ($_o-forms-radiocheckbox-default-size * 1.6) + $_o-forms-label-padding-left;
+			min-height: $_o-forms-radiocheckbox-default-size;
+			&:before {
+				position: absolute;
+				content: '';
+				background-color: oBrandGet('field-label-text');
+				top: 0;
+				left: 0;
+				width: $_o-forms-radiocheckbox-default-size * 1.6;
+				height: $_o-forms-radiocheckbox-default-size;
+				border-radius: ($_o-forms-radiocheckbox-default-size * 1.6);
+			}
+			&:after {
+				position: absolute;
+				content: '';
+				background-color: oColorsGetColorFor(page, background);
+				top: 2px;
+				left: 2px;
+				transition: left 0.075s ease-in-out;
+				width: $_o-forms-radiocheckbox-default-size - 4px;
```

Comment:
Where has this 4px come from?

Reply:
I couldn't remember 😬 Let me know whether the new variables are enough to clarify.